app/templates/main/userProfile.html

```diff
@@ -29,12 +29,12 @@ <h6 align="center">{{volunteer.email}}</h6>
 <div class="accordion" id="userProfile">
   <div class="accordion-item">
     <h3 class="accordion-header" id="headingOne">
-      {% set focus = "open" if visibleAccordion == "upcoming" or volunteer.username == g.current_user.username and not visibleAccordion =="programTable" else "collapsed" %}
+      {% set focus = "open" if not visibleAccordion or visibleAccordion == "upcoming" else "collapsed" %}
```

Comment:
Why did you modify the accordion?

Reply:
I did not edit it, something must've happened when pull development in. I don't think I meant to mess with the accordion at all. Will look into this.